### DISCUSSION TITLE
Pages for character selection menu

### DISCUSSION THREAD

**Discussion by MidyGamy** [FunkinCrew/Funkin#3322] [Ideas]
With the new update, playable characters have been added, but an issues will appear in the future: mods can add more than 7 characters to play...
So what I suggest is to make pages for mods folder characters, and let the 1st page only for base game characters

**Comment by Yasfunny** [comment]
It might get more noticed if you put it in Issues as an Enhancement

**Reply by Yasfunny** [reply]
personally i 100% agree

**Reply by MidyGamy** [reply]
are you spying on me dude?

**Reply by Lasercar** [reply]
What

**Comment by PSI3x** [comment]
![extraslotsmockup](https://github.com/user-attachments/assets/5ef1174c-1809-4d7d-92aa-35454fda6f6e)
I think that since modded slots will only utilize a couple extra characters, it could make more sense to have the menu scroll upwards to reveal extra modded characters. Most likely with some sort of prompt showing that you can scroll up.
